### DISCUSSION TITLE
Is it possible to mix disabled and tui=false?

### DISCUSSION THREAD

**Discussion by Nebucatnetzer** [F1bonacc1/process-compose#195] [Q&A] [answered]
I'm basically to implement this example: https://f1bonacc1.github.io/process-compose/launcher/#terminate-process-compose-once-given-process-ends

 I would like to adjust it so that I can run the tests only manually with: `process-compose up --tui=false tests`
However when I set `disabled: true` on the tests they don't run anymore.

**Comment by F1bonacc1** [comment, marked as answer]
Hi,

Yes, this is the expected behavior, the disabled processes are skipped during planning the start order.
Maybe something like namespaces will be a better fit for what you are trying to achieve.
Another option is to use multiple configuration files with overrides.

Please let me know if you considered that and if you need any help setting that up.

**Reply by Nebucatnetzer** [reply]
I had a look at namespaces, I’ve already forgotten why that didn’t work, probably because I would have to add a namespace to all processes🤔.
Multiple config files is difficult, I think, because it gets generated by devenv. 
I see what I can do.
